docs/dagster-university/pages/dagster-dbt/lesson-7/4-creating-the-manifest-during-deployment.md

```diff
@@ -18,12 +18,12 @@ To get our deployment working, we need to add a step to our GitHub Actions workf
 2. Open the `deploy.yml` file.
 3. Locate the `Checkout for Python Executable Deploy` step, which should be on or near line 38.
 4. After this step, add the following:
-    
+
```

Comment:
@erinkcochran87 do you know if these formatting changes are anything to worry about, is there an equivalent of make mdx-format for daggy u?